tests/e2escenarios/e2e_images_test.go

```diff
@@ -41,7 +41,10 @@ var _ = Describe("odo supported images e2e tests", func() {
 
 		// create the component
 		helper.CopyExample(filepath.Join("source", srcType), context)
-		helper.CmdShouldPass("odo", "create", cmpType, srcType+"-app", "--project", project, "--context", context, "--app", appName, "--min-memory", "400Mi", "--max-memory", "700Mi")
+		helper.CmdShouldPass("odo", "create", cmpType, srcType+"-app", "--project", project, "--context", context, "--app", appName)
```

Comment:
e2e test stands for broader user specific scenario. These validation can be done through UTs or integration test label. So please remove those lines. 

Ofcourse if we find a user specific scenario where we have such hard requirement of adding those validation then we will add these line for sure.

Reply:
If I don't trim the memory and cpu flags from here then this will fail.

Reply:
Ahh...got it. I did not realize it. Sorry for the false alarm.